types/index.d.ts

```diff
@@ -0,0 +1,183 @@
+/*!
+ * react-filepond v7.0.2
+ * A handy FilePond adapter component for React
+ *
+ * Copyright (c) 2020 PQINA
+ * https://pqina.nl/filepond
+ *
+ * Licensed under the MIT license.
+ */
+
+// Based on definitions by Zach Posten for React-Filepond <https://github.com/zposten>
+// Updated by FilePond Contributors
+
+import { Component } from 'react';
+import {
+  FileStatus,
+  ActualFileObject,
+  FilePondInitialFile,
+  FilePondOptions,
+} from 'filepond';
+
+export {};
```

Comment:
I had a comment in DefinitelyTyped explaining this line.  It seems like it might still be nice to have this explanation.  

```
// This line shuts off automatic exporting for this module
// We do this so that I can have internal types to this module that make
// understanding the type definition more clear, while at the same time only
// exporting the API that react-filepond exports.
```

Reply:
Entirely. I've removed the empty export. Thank you for your feedback, it's been really insightful.